include/zephyr/drivers/emul.h

```diff
@@ -124,4 +134,6 @@ const struct emul *emul_get_binding(const char *name);
  * @}
  */
 
+#include <emul_extern.h>
```

Comment:
Is it possible to `#include <generated/emul_extern.h>` for clarity? Do you know if we have just `include/` on the path?

Reply:
That doesn't work for me right now, so I guess we don't have just `include/` on the include path. Sounds like an opportunity for a future build system PR.

Reply:
Thanks for checking, marking this as resolved